src/test/java/tests/application/windowsApp/ApplicationTests.java

```diff
@@ -0,0 +1,58 @@
+package tests.application.windowsApp;
+
+import aquality.selenium.core.application.IApplication;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class ApplicationTests {
+
+    @BeforeMethod
+    public void setUp() {
+        AqualityServices.setApplication(null);
+    }
+
+    @AfterMethod
+    public void cleanUp() {
+        if (AqualityServices.isApplicationStarted()) {
+            AqualityServices.getApplication().getDriver().quit();
+        }
+    }
+
+    @Test
+    public void testShouldBePossibleToGetApplication() {
+        IApplication application = AqualityServices.get(IApplication.class);
+        Assert.assertNotNull(application, "should be possible to resolve app instance from aquality services");
+        Assert.assertEquals(AqualityServices.getApplication(), application,
+                "should return the same instance of app");
+    }
+
+    @Test
+    public void testShouldBePossibleToGetDriver() {
+        Assert.assertNotNull(AqualityServices.getApplication().getDriver(),
+                "should be possible get driver from the application");
+    }
+
+    @Test
+    public void testShouldBePossibleToSetImplicitWait() {
+        AqualityServices.getApplication().setImplicitWaitTimeout(1, TimeUnit.SECONDS);
```

Comment:
Would it better to wrap this into try\catch and use asser.fail in catch block?
I think test should have assert and some message in fail case.

Reply:
fixed